Command/Commands/GetWeatherForecastCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AppConfig;
+using AppConfig.Interfaces;
+using BL.CustomExceptions;
+using BL.Interfaces;
+using BL.Services;
+using Command.Interfaces;
+using DAL.Interfaces;
+using DAL.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Command.Commands
+{
+    public class GetWeatherForecastCommand : ICommand
+    {
+        private static IConfig _config = new Config();
```

Comment:
Remove "= new Config()" and write it "_config = new Config()" in a GetWeatherForecastCommand

Reply:
Fixed